mlflow/pyfunc/stdin_server.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import argparse
+import json
+import mlflow
+from mlflow.pyfunc import scoring_server
+import logging
+from pathlib import Path
+
+_logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--model-uri")
+args = parser.parse_args()
+
+_logger.info("Loading model from %s", args.model_uri)
+model = mlflow.pyfunc.load_model(args.model_uri)
+input_schema = model.metadata.get_input_schema()
+_logger.info("Loaded model")
+
+_logger.info("Waiting for request")
+for line in sys.stdin:
```

Comment:
To make the server robust, we should check unexpected input and catch exception raised from the for-loop body

Reply:
Yes, I'll make `mlflow/pyfunc/stdin_server.py` more robust.